homeassistant/components/simplefin/strings.json

```
@@ -0,0 +1,35 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Please enter either a Claim Token or an Access URL.",
+        "data": {
+          "api_token": "Claim Token or Access URL"
+        }
+      },
+      "reauth_confirm": {
+        "description": "Please enter either a Claim Token or an Access URL.",
+        "data": {
+          "api_token": "Claim Token or Access URL"
+        }
+      }
+    },
+    "error": {
+      "auth_error": "Authentication failed. This could be because access has been revoked or if the credentials are incorrect",
```

```suggestion
      "invalid_auth": "Authentication failed. This could be because access has been revoked or if the credentials are incorrect",
```
Can we also make the text a bit more concrete?